tests/workingWithAPI.spec.ts

```diff
@@ -26,15 +26,20 @@ test('mocking API request', async ({page}) => {
 
     await pm.onOwnersPage().validateOwnerCount(2)
 
-    // Capturing data from the testOwners JSON file, which contains information to be displayed on the Owners Page
-    const testOwnerFullName = `${testOwners[0]['firstName']} ${testOwners[0]['lastName']}`
-    
+    // Extracting full name from API Mock to click on link in 
+    const testOwnerFullName = `${testOwners[0]['firstName']} ${testOwners[0]['lastName']}`    
```

Comment:
I would suggest this to make a little bit nicer.
Instead of reading the full object every time, you can create a constant first that will represent your test owner for this test.
Just by index:
`const owner = testOwners[0]`
or by using more nicer find() method and get your owner by any property, for example by ID:
`const owner = testOwners.find(owner => owner.id === 1);`

Then it's easier to work with this object and it looks nicer as well:
```const testOwnerFullName = `${owner.firstName} ${owner.lastName}` ```

"dot notation" is easier to read in JSON objects than "bracket notation" as you can see. You can refactor the rest of your test accordingly.

Reply:
I've learned some Javascript here.